src/components/panel/panel.spec.js

```diff
@@ -1880,6 +1880,31 @@ describe('$mdPanel', function() {
         });
       });
 
+      it('should have assigned the actual position by the time the offset' +
+        'methods have been called', function() {
+          var hasActualPosition = false;
+
+          var obj = {
+            getOffsetX: function(position) {
```

Comment:
I initially tried something similar, but the problem with this approach is that it's testing against the final object, which means that the test will pass even without the current changes.